jobs/base_job_sample.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env python
+
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import random
+import string
+
+# [START instantiate]
+from googleapiclient.discovery import build
+from googleapiclient.errors import HttpError
+
+client_service = build('jobs', 'v2')
+
+# [END instantiate]
+
+
+# [START basic_job]
+def generate_job_with_required_fields(company_name):
```

Comment:
I feel that this can be a simple `job` variable as well.

Reply:
Just to distinguish this function from other generate job function such as generate_job_with_custom_attribute